### PR TITLE
Increased maximum excel size to 60MB

### PR DESCRIPTION
**Pull request by kaleween**

### CODE REVIEW

File: handlers/reader.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 func ReadExcelFileByLine(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	file, _, _ := r.FormFile("file")
-	r.ParseForm()
+	r.ParseMultipartForm(60)
```
Comment (Jerome1337) [inline handlers/reader.go]: should be `60 << 20` (means 60mb)

File: handlers/reader.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func ReadExcelFileByLine(w http.ResponseWriter, r *http.Request) {
 func ReadExcelFileByColumn(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	file, _, _ := r.FormFile("file")
-	r.ParseForm()
+	r.ParseMultipartForm(60)
```
Comment (Jerome1337) [inline handlers/reader.go]: should be `60 << 20` (means 60mb)

**Review by Jerome1337 — CHANGES_REQUESTED** [review]

File: handlers/reader.go (hunk 1 above)
Comment (kaleween) [reply handlers/reader.go]: done

File: handlers/reader.go (hunk 2 above)
Comment (kaleween) [reply handlers/reader.go]: done

**Review by rbaraffe — APPROVED** [review]

**Review by Jerome1337 — APPROVED** [review]

**Comment by Jerome1337** [comment]
Closed in favor of #47

**Pull request closed without merging**